banyand/series/schema/sw/trace_series.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
@wu-sheng @Fine0830  This's the pre-defined schema for writing and querying as we discussed in the write API PR.